CHANGES_NEXT_RELEASE

```diff
@@ -12,3 +12,4 @@
 - [BUG] Add OrionDynamoDBSink configuration template in cygnus_agent_conf.md (#762)
 - [BUG] Add OrionPostgreSQLSink configuration template in cygnus_agent_conf.md (#769)
 - [BUG] Add missing sink types in agent.conf.templates (#773)
+- [BUG] Fix wrong data_model configured in agent.conf.template (#774)
```

Comment:
I would say "Fix wrong data_model configured for OrionPostgreSQLSink in agent.conf.template (#774)"